doc/api/process.md

```diff
@@ -745,7 +745,7 @@ process.exit(1);
 The shell that executed Node.js should see the exit code as `1`.
 
 It is important to note that calling `process.exit()` will force the process to
-exit as quickly as possible *even if there are still asynchronous operations 
```

Comment:
well... two trailing spaces in markdown means line break. though it's one of the most stupid things I've ever heard of, so don't worry about it.